src/main/resources/swagger/api-docs.yaml

```diff
@@ -1986,6 +1986,31 @@ paths:
           required: false
           schema:
             $ref: '#/definitions/AttributeUpdateOperation'
+
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/{entityName}/evaluate:
+      post:
+        tags:
+          - Entities
+        operationId: getEntity
+        summary: Get entity in a workspace
+        parameters:
+          - $ref: '#/parameters/workspaceNamespaceParam'
+          - $ref: '#/parameters/workspaceNameParam'
+          - $ref: '#/parameters/entityTypeParam'
+          - $ref: '#/parameters/entityNameParam'
+          - in: body
+            description: Expression
+            name: expression
+            required: true
+            type: string
+        responses:
+          200:
+            description: Success
+          404:
```

Comment:
... and 403 is missing from Rawls Swagger so we should update that too!  @vkumra-broad could you add that to https://github.com/broadinstitute/rawls/pull/739 ?